python/graphstorm/model/utils.py

```diff
@@ -68,6 +68,8 @@ def save_model(model_path, gnn_model=None, embed_layer=None, decoder=None):
         model_states['decoder'] = decoder.state_dict()
 
     os.makedirs(model_path, exist_ok=True)
+    # [04/16]: Assume this method is called by rank 0 who can perform chmod
+    os.chmod(model_path, 0o777)
```

Comment:
Can we remove the exec permission?

Reply:
I tested and the least privilege is 767. See my reply to Theo's suggestion.